content/docs/releases/v1.12.0.md

```diff
@@ -0,0 +1,389 @@
+
++++
+short="v1.12.0"
+title = "Release v1.12.0"
+date="2024-01-24 00:00:00 +0000 UTC"
++++
+
+
+# NSM Release v1.12.0 
+
+
+NSM v1.12.0 has been tested on:
+- [kind](https://github.com/networkservicemesh/integration-k8s-kind/actions?query=branch%3Arelease%2Fv1.12.0+)
+- [GKE](https://github.com/networkservicemesh/integration-k8s-gke/actions?query=branch%3Arelease%2Fv1.12.0+)
+- [AKS](https://github.com/networkservicemesh/integration-k8s-aks/actions?query=branch%3Arelease%2Fv1.12.0+)
+- [AWS/AWS IPv6](https://github.com/networkservicemesh/integration-k8s-aws/actions?query=branch%3Arelease%2Fv1.12.0+)
+- [Bare metal in Equinix Metal](https://github.com/networkservicemesh/integration-k8s-packet/actions?query=branch%3Arelease%2Fv1.12.0+)
+- [Interdomain GKE/AWS/AKS](https://github.com/networkservicemesh/integration-interdomain-k8s/actions?query=branch%3Arelease%2Fv1.12.0+)
+
+## Changes since last release
+
+
+
+### `begin` can't do `Unregister` requests without `Register` requests
```

Comment:
It's been made in v1.11.0. I think we should delete it from v1.12.0

Reply:
Fixed